modules/core/02-client/keeper/grpc_query_test.go

```diff
@@ -163,10 +163,10 @@ func (suite *KeeperTestSuite) TestQueryClientStates() {
 			ctx := sdk.WrapSDKContext(suite.chainA.GetContext())
 
 			res, err := suite.chainA.QueryServer.ClientStates(ctx, req)
-
 			if tc.expPass {
 				suite.Require().NoError(err)
 				suite.Require().NotNil(res)
+				suite.Require().Equal(uint64(len(expClientStates)), res.Pagination.Total)
```

Comment:
Really a super nit, but my consistency freak would write the check the same way as in this [other test](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel/keeper/grpc_query_test.go#L197). And maybe also put it as the last one (after `suite.Require().Equal(expClientStates.Sort(), res.ClientStates)`).

```suggestion
				suite.Require().Equal(len(expClientStates)), int(res.Pagination.Total))
```